src/main/scala/cromwell/engine/backend/jes/authentication/JesAuthentication.scala

```diff
@@ -20,7 +34,12 @@ object ProductionJesConnection {
  */
 trait JesAuthentication { self: JesConnection =>
 
-  def authenticated[A](f: JesInterface => A) = f(jesConnection)
+  def authenticateAsCromwell[A](f: JesInterface => A) = f(jesCromwellConnection)
+
+  /**
+   * Important note: Will default back to cromwell authentication if the configuration for user authentication has not been set or if the refreshToken has been supplied.
```

Comment:
Also it's actually the project you request compute in that determines who is getting billed. So for us to get billed instead of them we would have to receive our google project in the workflow options instead of the user's.